modules/base/tests/playwright/ServerRoutes.spec.js

```diff
@@ -55,3 +55,43 @@ test('request on different path doesnt invoke server route', async ({ page }) =>
     },
   )
 })
+
+test('server route with path prefix', async ({ page }) => {
+  await makeProject(
+    {
+      modules: [
+        '@storefront-x/base',
+        '@storefront-x/vue',
+        '@storefront-x/vue-i18n',
```

Comment:
Why is vue and especially vue-i18n module enabled for testing server routes? Thats definetelly not needed.